app-arch/rpm/files/rpm-4.18.1-musl-nls.patch

```diff
@@ -0,0 +1,39 @@
+From da222f0d82721d17e57d491d2749a4bc44754b9d Mon Sep 17 00:00:00 2001
```

Comment:
Sent this upstream? Please do and include a link here.

Reply:
Added links. It has changed slightly from the patch that was merged upstream for compatibility with previous version.